MessengerOnFileForLinux/UserService/src/RegisterUser.cpp

```diff
@@ -93,44 +93,45 @@ bool RegisterUser::registerNewUser() const
         return false;
     }
 
+    std::array<std::string, 2> passwords;
     bool isPasswordSetCorrectly = false;
     while (!isPasswordSetCorrectly)
```

Comment:
Może zamiast tego array to dwa stringi by było to czytelniejsze?

Reply:
Do przemyślenia, ale na razie zostawiam array. Przy refaktorze możliwe że będzie wydzielona funkcja która raz odpytuje użytkownika o hasło i zwraca stringa, a nie tak jak teraz.

Reply:
zrób proszę taska na refactor i wpisz to do listy zadań w takim razie